package.json

```diff
@@ -173,7 +173,6 @@
     "@open-draft/logger": "^0.3.0",
     "@open-draft/until": "^2.0.0",
     "is-node-process": "^1.2.0",
-    "jsdom": "^26.0.0",
```

Comment:
Well, we still need `jsdom` for integration tests! Could you please move it to `dependencies `instead? Thanks.

Reply:
are you sure? they seem to mostly work without jsdom. At least I get the same errors with and without jsdom... That's why I removed the dependency completely instead of moving to devDependency. But happy to add it back if you think about it strongly :-)

Reply:
We use jsdom for testing of the library. So yes, we depend on it while you don't.

Reply:
Yes I understood this. But when after removing it, strangely I could still run all tests properly. I'm curious to know which tests need jsdom.

Anyway I pushed the commit adding jsdom to devDependencies. This also updates it to 26.1.0.

Reply:
All XHR tests use JSDOM. If they ran, that must've been some issue or caching.

Reply:
After removing it fully, when running `pnpm why jsdom` I still see it.

So I wondered if the peer dependencies were somehow installed automatically by pnpm.
And boom I see `autoInstallPeers: true ` is in `pnpm-lock.yaml`, so that's the reason: jsdom was present thanks to being a peer dependency of `vitest`.

However having it in `package.json` is useful because you're in control of the exact version, while `vitest`'s peer dependency is `*`.

I'm glad I got to the bottom of it :-)